### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by byroot**
WIP

### CODE REVIEW

File: documentation/shipit.yml.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  unlock
+
+dependencies
+  bundler
+
+### review
+
+<code>review.checklist</code>
+
+Array of strings. Can contain and HTML tags. Users are required to check each of them before being able to deploy.
```
Comment (rafaelfranca) [inline documentation/shipit.yml.md]: I think there is an extra `and` there. It should be `Can contain HTML tags.`

File: documentation/shipit.yml.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+curl --silent https://app.shopify.com/services/ping/version
+
+### deploy
+
+<code>deploy.override</code>
+
+Array of shell commands required to deploy the application. Shipit will try to infer it from the repository structure, but you can change the default inference.
+
+exec ./script/multi_deploy.rb bundle exec cap $ENVIRONMENT deploy
```
Comment (rafaelfranca) [inline documentation/shipit.yml.md]: Is this the default inference? If so, it may be good to make it explicit.

**Comment by byroot** [comment]
Oh I forgot about that PR. I should rebase from master since this was initially an extraction gimme a minute.

**Comment by byroot** [comment]
Yeah It's what I though. That file basically became the README. Closing this.

File: documentation/shipit.yml.md (hunk 1 above)
Comment (byroot) [inline documentation/shipit.yml.md]: Good catch. I'll update master.

File: documentation/shipit.yml.md (hunk 2 above)
Comment (byroot) [inline documentation/shipit.yml.md]: No it's a custom one Rory grabbed from one of our repos. The default inference is based on multiple heuristics: https://github.com/Shopify/shipit-engine/tree/master/app/models/deploy_spec

**Pull request closed without merging**